<launch>
    <arg name="map" />

    <include file="$(find racer)/launch/localization.launch">
        <arg name="map" value="$(arg map)"/>
    </include>

    <node pkg="racer" name="circuit_analysis" type="circuit_node" output="screen" />
    <node pkg="racer" name="planning" type="planning_node" output="screen" />
    <node pkg="racer" name="following" type="following_node" output="screen">
        <param name="strategy" value="geometric" />
    </node>
</launch>
